psd-web/app/policies/investigation_policy.rb

```diff
@@ -36,6 +36,16 @@ def view_non_protected_details?(user: @user, private: @record.is_private)
     false
   end
 
+  def view_protected_details?(user: @user)
+    # Is the user a member of the case owner's team?
+    return true if @record.owner.present? && (@record.owner&.team == user.team)
```

Comment:
```suggestion
    return true if @record.owner&.team == user.team
```
I think this is enough. as `user` **must** have a team. Worst case scenario this should always be false `nil == instance_of(Team)`

Reply:
I think the rights needed are the same as on L7. We could either reuse that, or use the same code?

We don't need to test the owner - it's purely about whether the current user's team is in `record.teasm_with_access` I think.

Reply:
This was copied from the preceding `view_non_protected_details?` policy for consistency but  I'll look at this.